packages/expression-parser/src/expression-parser/customFunctions.js

```diff
@@ -30,9 +32,12 @@ const translate = (value, translations) => {
 
 const date = (...argumentList) => new Date(...argumentList);
 
+const dateUtils = () => fns;
+
 export const customFunctions = {
   avg: average,
   firstExistingValue,
   translate,
   date,
+  dateUtils,
```

Comment:
Tried to spread `...fns` in here but it has other functions as we do, such as `format`.

Reply:
Honestly I think namespacing behind `dateUtils` is a better move anway 👍 I've been thinking we may want to add namespacing to functions in expressions for a while, eg. `math.sum()`, `entity.codeToId()` or the like. Otherwise over time things will get messy :S

Reply:
That is a great idea! Actually it is super easy to support that. See the latest changes.